sdk/java/src/main/java/com/gojek/feast/FeastClient.java

```diff
@@ -39,8 +40,8 @@
 
   private static final int CHANNEL_SHUTDOWN_TIMEOUT_SEC = 5;
 
-  private final ManagedChannel channel;
-  private final ServingServiceGrpc.ServingServiceBlockingStub stub;
+  private ManagedChannel channel;
```

Comment:
I think you should keep these as is (`final`). if you want to repalce the channel and stub with mocks for testing purposes, you might want to make a protected all args constructor instead.

Reply:
I only need to mock `channel`. Mocking `stub` is not possible as its a `final` class.

Reply:
Corrected.